SignalServiceKit/src/Loki/Utilities/LokiPushNotificationManager.swift

```diff
@@ -70,4 +73,24 @@ final class LokiPushNotificationManager : NSObject {
             print("[Loki] Couldn't register device token.")
         })
     }
+    
+    @objc(acknowledgeDeliveryForMessageWithHash:expiration:hexEncodedPublicKey:)
+    static func acknowledgeDeliveryForMessage(with hash: String, expiration:Int, hexEncodedPublicKey: String) {
```

Comment:
Minor thing, but it'd be a bit more Swifty to call this `acknowledgeDelivery(forMessageWithHash:expiration:hexEncodedPublicKey:)`